docs/next.config.js

```diff
@@ -51,7 +51,7 @@ module.exports = withNextPluginPreval({
   },
 
   // These redirects are because of the IA change from previous docs
-  redirects() {
+  async redirects() {
```

Comment:
For my understanding, why `async`?

Reply:
`Next` config requires it to be async anyway and we didn't configure it correctly in the first place :( <- https://nextjs.org/docs/api-reference/next.config.js/redirects. But yea, `Next` doesn't explain much about why it should be async.